src/sentry/issues/issue_velocity.py

```diff
@@ -24,16 +24,21 @@
     Request,
 )
 
-from sentry.models.project import Project
 from sentry.snuba.dataset import Dataset, EntityKey
-from sentry.tasks.post_process import locks
 from sentry.utils import json
 from sentry.utils.locking import UnableToAcquireLock
+from sentry.utils.locking.manager import LockManager
 from sentry.utils.redis import redis_clusters
-from sentry.utils.snuba import raw_snql_query
+from sentry.utils.services import build_instance_from_options
+
+if TYPE_CHECKING:
+    from sentry.models.project import Project
+
 
 logger = logging.getLogger(__name__)
 
+locks = LockManager(build_instance_from_options(settings.SENTRY_POST_PROCESS_LOCKS_BACKEND_OPTIONS))
```

Comment:
This seems fine, but wondering if there's any reason we do this rather than use the post process one?

Reply:
i originally used the post process one but got a circular dependency error when i ran tests

Reply:
never mind i stand corrected, pytest just gaslit me, will change back to use the post process locks